client/src/stories/Input.stories.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import InputField, { InputContainer } from '../components/Input';
+import EmailAtIcon from '../assets/email-at.svg';
+import PasswordIcon from '../assets/password.svg';
+import UserIcon from '../assets/user.svg';
+import LocationIcon from '../assets/pin.svg';
+import AgeIcon from '../assets/age.svg';
+import EmailIcon from '../assets/email.svg';
+import PhoneIcon from '../assets/phone.svg';
+import UserGroupIcon from '../assets/user-group.svg';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Inputs',
+};
+
+export const mailInput = () => (
+  <InputContainer>
+    <img src={EmailAtIcon} alt="email" />
+    <InputField type="email" placeholder="E-mail" onChange={action('write')} />
+  </InputContainer>
+);
+
+export const passwordInput = () => (
+  <InputContainer>
```

Comment:
You use multiple and the same Components to build your input.
I would highly suggest building one Input and just give it different props to have diversity.
for example:
```js
const Input = ({ src, alt, placeholder <ect.> }) => {
<InputContainer>
    <img src={src} alt={alt} />
    <InputField type="text" placeholder={placeholder} onChange={action('write')} />
  </InputContainer>
}
```

```js
const UserNameInput = () => {
<Input src={UserIcon} alt="name" placeholder="Name"  <ect.> />
}
```

Just a suggestion 😄

Reply:
Thanks for the helpful hint, it was not easy but I have now been able to combine it into one component as input field 👍🏼thanks!